File: selection sort/selection_sort.java
import java.util.Scanner;

class selection_sort {
    public static void main(String args[]) {

        Scanner sacn = new Scanner(System.in);

        void selectionSort(int arr[]) {

            int min_index = 0;
            int temp;
            int n = arr.length;
            for (int i = 0; i < n - 1; i++) {
                min_index = i;
                for (int j = i; j < n; j++) {
                    if (arr[j] < arr[min_index]) {
                        min_index = j;
                    }
                }
                temp = arr[i];
                arr[i] = arr[min_index];
                arr[min_index] = temp;
            }

            return 0;
        }

        System.out.println("Enter length of string :");
        int n = sacn.nextInt();

        int[] arr = new int[n];
        for (int i = 0; i < n; i++) {
            arr[i] = sacn.nextInt();
        }

        selectionSort(arr);

        for (int num : arr) {
            System.out.print(num + " ");
        }
    }
}

File: selection sort/selection_sort.py
arr = list(map(int, input("Enter a list of numbers with space :").strip(" ").split(" ")))

def selection_sort(arr):
    #return a array if length is leass than one
    if(len(arr) <= 1):
        return arr
    
    for i in range(len(arr)-1):
        min_index = i
        for j in range(i+1, len(arr)):
            if(arr[j] < arr[min_index]):
                min_index = j
        temp = arr[i]
        arr[i] = arr[min_index]
        arr[min_index] = temp
    
    return arr

print(arr)  
print(selection_sort(arr))